sdk/compute-budget-instruction/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "solana-compute-budget-instruction"
```

Comment:
nit: can we rename this `solana-compute-budget-interface`? Even though there isn't any state for the program, it'll be more consistent to call it an "interface" like with the other native programs